.github/workflows/add-action-project.yml

```diff
@@ -0,0 +1,26 @@
+name: Add new issues to nim-waku project board
+
+on:
+  issues:
+    types: [opened]
+  pull_request:
+    types: [opened]
+
+jobs:
+  add-new-issue-to-new-column:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Add issue to column "New"
+        if: ${{ !!github.event.issue }}
+        uses: alex-page/github-project-automation-plus@v0.6.0
+        with:
+          project: nim-waku
+          column: New
+          repo-token: ${{ secrets.GH_ACTION_PROJECT_MGMT }}
+      - name: Add pull request to column "In Progress"
```

Comment:
Not sure why "In Progress" is not working - perhaps difference in capitalisation?

Reply:
Fixed, also changed around a bit